client/my-sites/stats/stats-post-detail/index.jsx

```diff
@@ -162,7 +162,7 @@ class StatsPostDetail extends Component {
 						action={ translate( 'Get more traffic!' ) }
 						actionURL="https://wordpress.com/support/getting-more-views-and-traffic/"
 						actionTarget="blank"
-						illustration="/calypso/images/stats/illustration-stats.svg"
+						illustration="calypso/assets/images/stats/illustration-stats.svg"
```

Comment:
Could this svg be imported on the top of the file?

Reply:
Likewise with this change. Visiting the My Home page triggers a 404 for this SVG asset.